manifests/init.pp

```diff
@@ -276,6 +277,7 @@
   Stdlib::Absolutepath $inetrc_config_path                         = $rabbitmq::params::inetrc_config_path,
   Boolean $ssl_erl_dist                                            = $rabbitmq::params::ssl_erl_dist,
   Optional[String] $rabbitmqadmin_package                          = $rabbitmq::params::rabbitmqadmin_package,
+  [Array] $archive_options                                         = $rabbitmq::params::archive_options,
```

Comment:
Take out the leading parens when you get rid of `Optional`, so `Array $archive_options`

Reply:
yep - sorry about that typo